pkg/acsengine/defaults.go

```diff
@@ -384,7 +384,7 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 		switch o.KubernetesConfig.NetworkPolicy {
 		case NetworkPluginAzure:
 			o.KubernetesConfig.NetworkPlugin = NetworkPluginAzure
-			o.KubernetesConfig.NetworkPolicy = DefaultNetworkPolicy
+			o.KubernetesConfig.NetworkPolicy = NetworkPolicyAzure
```

Comment:
What is the reason for changing the default here? This might break backwards compatibility so we'll have to do some extensive testing if this is something we want. cc: @jackfrancis

Reply:
Previously we don't have network policy support in Azure CNI, now we just released it. Therefore we want to set default NetworkPolicy to azure.